test/ServerlessClamscan.test.ts

```diff
@@ -167,6 +167,24 @@ test('expect VirusDefsBucket to not have access logging enabled', () => {
   });
 });
 
+test('expect reserved capacity prop to set Lambdas reserved capacity', () => {
+  const stack = new Stack();
+  new ServerlessClamscan(stack, 'default', {
+    reservedCapacity: 100,
+  });
+  expect(stack).toHaveResourceLike('AWS::Lambda::Function', {
+    ReservedConcurrentExecutions: 100,
+  });
+});
+
+test('expect reserved capacity to be optional', () => {
+  const stack = new Stack();
+  new ServerlessClamscan(stack, 'default', {});
+  expect(stack).not.toHaveResourceLike('AWS::Lambda::Function', {
+    ReservedConcurrentExecutions: 100,
```

Comment:
```suggestion
    ReservedConcurrentExecutions: anything(),
```

use and import the `anything` matcher from `@aws-cdk/assert`

Reply:
Nice!